### PR TITLE
LayoutCard: Use the colormap too when rendering, if available

### PR DESCRIPTION
**Pull request by algernon**
If we're connected to a keyboard that has Colormap support, render the layers with the colormap applied, too.

Screenshot from 2022-05-26 13-16-19

### CODE REVIEW

**Comment by obra** [comment]
Since the primary point of the layout card feature is printability and most folks don't have color printers at home, I don't think showing color maps is the right choice here.

**Comment by algernon** [comment]
Can we make it an option then? I find it useful to look at it - with colors - on a second screen, and the colors are a great guide in that case.

**Comment by obra** [comment]
As long as it lives in preferences, sure :)

**Comment by obra** [comment]
More importantly, preferences is more correct. :)

It and theme both eventually belong in a category called “User Interface” or something like it.

**Comment by algernon** [comment]
Pushed an update:

Screenshot from 2022-05-26 21-19-12

Works as one would expect: toggling it on will add the colormap to the layout card, otherwise it will remain monochrome. Defaults to monochrome.

I'm a bit conflicted about the setting, and part of me wants to flip it, and have it an `[x] Use monochrome layout cards`, and thus have it enabled by default. But the wording of that still doesn't feel exactly right to me.

File: src/renderer/screens/Preferences/Basic.js [hunk 1]
```diff
@@ -55,6 +62,10 @@ function BasicPreferences(props) {
     }
   };
 
+  useEffectOnce(() => {
```
Comment (obra) [inline src/renderer/screens/Preferences/Basic.js:65]: We shouldn't need useEffectOnce here.  uEO is a "weird" thing that's kind of not the React Way.

why doesn't regular useEffect work?

Comment (algernon) [reply src/renderer/screens/Preferences/Basic.js:65]: I suppose regular useEffect would work here too.

**Comment by algernon** [comment]
Rebased on top of master, and replaced `useEffectOnce` with `useEffect`.

**Pull request merged**